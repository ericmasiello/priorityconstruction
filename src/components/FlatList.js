import styled from 'styled-components';
import List, { ListItem } from './List';

const FlatList = styled(List)`
  display: flex;
`;

FlatList.Item = styled(ListItem)`
  &:not(:last-child) {
    margin-right: 0.5rem;
  }
`;

export default FlatList;
