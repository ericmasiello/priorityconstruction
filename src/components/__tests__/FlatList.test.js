import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import FlatList from '../FlatList';

it('should render <FlatList>', () => {
  const component = renderer.create(<FlatList />);

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

it('should render <FlatList.Item>', () => {
  const component = renderer.create(<FlatList.Item />);

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
