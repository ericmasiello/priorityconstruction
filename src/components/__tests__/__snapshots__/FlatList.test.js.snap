// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render <FlatList.Item> 1`] = `
.c0:not(:last-child) {
  margin-bottom: 0.5rem;
}

.c0:not(:last-child) {
  margin-right: 0.5rem;
}

<li
  className="c0"
/>
`;

exports[`should render <FlatList> 1`] = `
.c0 {
  list-style-type: none;
  margin: 0;
  padding: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<ul
  className="c0"
/>
`;
